### PR TITLE
SPARKC-345 Add ttl per row insertion to Sql write

### PR DESCRIPTION
**Pull request by alexliu68**

### CODE REVIEW

File: spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/WriteConf.scala [hunk 1]
```diff
@@ -122,6 +122,12 @@ object WriteConf {
       | per single core in MB/s. <br> Limit this on long (+8 hour) runs to 70% of your max throughput
       | as seen on a smaller job for stability""".stripMargin)
 
+  val TtlParam = ConfigParameter[Int] (
```
Comment (RussellSpitzer) [inline spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/WriteConf.scala]: I think this should be TTLParam, Ttl just looks weird to me?

File: spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/WriteConf.scala [hunk 2]
```diff
@@ -122,6 +122,12 @@ object WriteConf {
...
+    name = "spark.cassandra.output.ttl",
+    section = ReferenceSection,
+    default = 0,
+    description = """Time to live in seconds for the columns per insertion""".stripMargin)
```
Comment (RussellSpitzer) [inline spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/WriteConf.scala]: Description may want to say, """Time To Live(TTL) assigned to writes to Cassandra"""

Comment (RussellSpitzer) [inline spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/WriteConf.scala]: And maybe ( A value of 0 means no TTL)

File: spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/WriteConf.scala (hunk 1 above)
Comment (RussellSpitzer) [inline spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/WriteConf.scala]: My gut is also telling me this should be [Option[TTLOption]] with a default of None ... or possibly  we add another member to TTLOption which is None.

Comment (pkolaczk) [inline spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/WriteConf.scala]: yup, we have "TTL" everywhere else, so let's be consistent

**Comment by pkolaczk** [comment]
+1

**Comment by pkolaczk** [comment]
@alex please squash and merge

**Comment by josephykwang** [comment]
awesome! which version of connector has this feature?

**Comment by RussellSpitzer** [comment]
1.6.0 :) You can always check the Jiras
https://datastax-oss.atlassian.net/browse/SPARKC-345

**Pull request merged**